_develop/deploy-tips.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: learn
+description: Blockstack app mining documentation
+permalink: /:collection/:path.html
+---
+# How to deploy a Blockstack dApp
+{:.no_toc}
+
+Blockstack applications are a type of web application that uses Blockstack's technology to authenticate users. As such, Blockstack apps tend to be simple in design and operation, since in many cases, they don’t have to host anything besides the application’s assets.
+
+ * TOC
+ {:toc}
+
+## Where to deploy your application
+
+Before users can interact with your application, you must deploy it on a server that is accessible over the internet. Deploying requires that you:
+
+* Configure or customize the files in the `public` directory.
+* Build your application site for deployment.
+* Copy your generated application files to your production server. 
+
+If you first populated your application with the Blockstack application generator, your application contains the starting blocks for configuring, building, and deploying your app. For example, the React template builds out a scaffolding with the following building blocks.
+
+<table class="uk-table">
+  <tr>
+    <th>File or Directory</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>node_modules/react-scripts</code></td>
+    <td>A set of scripts for that helps you kick off React projects without configuring, so you do not have to set up your project by yourself.</td>
+  </tr>
+  <tr>
+    <td><code>package.json</code></td>
+    <td>Contains a scripts section that includes a reference to the react-scripts, which are a dependency. This script creates a <code>build</code> directory containing your files for deployment.</td>
+  </tr>
+  <tr>
+    <td><code>public/favicon.ico</code></td>
+    <td>An example shortcut icon.</td>
+  </tr>
+  <tr>
+    <td><code>public/index.html</code></td>
+    <td>An entry page for an application.</td>
+  </tr>
+  <tr>
+    <td><code>public/manifest.json</code></td>
+    <td>A JSON file that describes your web application to the browser.&nbsp;&nbsp;</td>
+  </tr>
+  <tr>
+    <td><code>cors</code></td>
+    <td>Contains example deployment files for cross-origin request configuration.</td>
+  </tr>
+</table>
+
+If you use the generator to build Javascript or Vue scaffolding, your project configuration files will be different. 
+
+Regardless of which scaffolding you use, you must customize and extend this basic scaffolding as needed by your application. For example, you may want to add more properties to the `manifest.json` file. Since every application is different, Blockstack cannot give you specific instructions on how to do this. The steps you take are specific to your application.
+
+## Blockstack Authentication and deployment
```

Comment:
I think this section should mention that the included configs in the `cors` directory are specifically for making the `manifest.json` file accessible to the outside world (i.e. to the blockstack browser). If users are using something other than Netlify or Firebase, they will specifically need to configure CORS on their service of choice to include the  following headers when serving `manifest.json`:

```
Access-Control-Allow-Origin: *
Access-Control-Allow-Headers: "X-Requested-With, Content-Type, Origin, Authorization, Accept, Client-Security-Token, Accept-Encoding"
Access-Control-Allow-Methods: "POST, GET, OPTIONS, DELETE, PUT"```

Reply:
done. Thanks for the review! @reedrosenbluth